src/params/progression.jl

```diff
@@ -50,7 +52,13 @@ function sample_severity(rng::AbstractRNG, age::Real)
   else;             dist = severity_dists[6]
   end
 
-  rand(rng, dist) |> Severity
+  severity = rand(rng, dist) |> Severity
+  severity_int = severity |> UInt8
+  if severity > Asymptomatic && vaccinated(immunity) && rand(rng) < vaccination_effectiveness[severity_int]
```

Comment:
natural immunization should also have an effect here

Reply:
oki, but the same effect?